pkg/scalers/graphite_scaler.go

```diff
@@ -194,7 +194,8 @@ func (s *graphiteScaler) ExecuteGrapQuery(ctx context.Context) (float64, error)
 	}
 
 	// https://graphite-api.readthedocs.io/en/latest/api.html#json
-	datapoint := result[0].Datapoints[0][0]
+	latestDatapoint := len(result[0].Datapoints) - 1
```

Comment:
I am curious, is there a possibility that `len(result[0].Datapoints) == 0` ? If so, we should check it here.

Reply:
This should be impossible, but I added a check anyway.